### DISCUSSION TITLE
Unable to run the docker-compose.yml file

### DISCUSSION THREAD

**Discussion by HRTshazab** [dec0dOS/zero-ui#72] [General]
When im trying to implement it using the docker method, initially it throws an error that 

**ERROR: Version in "./docker-compose.yml" is unsupported. You might be seeing this error because you're using the wrong Compose file version. Either specify a supported version (e.g "2.2" or "3.3") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.**

Later updated the docker-compose version tried to run again. Now getting different issue.....
root@instance-2:~# docker -v
Docker version 20.10.14, build a224086
root@instance-2:~# docker-compose -v
Docker Compose version v2.5.0
root@instance-2:~# docker-compose pull
[+] Running 0/0
 ⠋ zero-ui Pulling                                                                                                                                                                         0.0s
 ⠋ https-proxy Pulling                                                                                                                                                                     0.0s
 ⠋ zerotier Pulling                                                                                                                                                                        0.0s
WARNING: Some service image(s) must be built from source by running:
    docker compose build zero-ui zerotier
error getting credentials - err: exit status 1, out: `Cannot autolaunch D-Bus without X11 $DISPLAY`

**Comment by dec0dOS** [comment]
Hello, @HRTshazab. Seems that something is really wrong with your docker installation.

**Reply by HRTshazab** [reply]
as per the discussion ive removed the docker and re-installed again still facing the same issue with the docker regarding version number.
**ERROR: Version in "./docker-compose.yml" is unsupported. You might be seeing this error because you're using the wrong Compose file version. Either specify a supported version (e.g "2.2" or "3.3") and place your service definitions under the services key, or omit the version key and place your service definitions at the root of the file to use version 1**

It only worked when i changed the docker version in the docker-compose.yml.
Is it okay to change the version number......?

**Reply by dec0dOS** [reply]
Yes, that is fine. I should have used the older default version for older docker and docker-compose systems.

**Reply by n9yty** [reply]
I am trying this on a new install of docker/docker-compose and also am seeing the same problem. This is on a ubuntu 20.04 install. The current docker-compose in focal seems to be 1.25.0-1.

**Reply by HRTshazab** [reply]
By changing the version from3.9 to 3.3 in docker file it fixed @n9yty
